.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,78 @@
+name:  🐞 Bug Report
+description: Create a report to help us improve
+labels: ["bug", "new issue"]
```

Comment:
I noticed we're missing a 'new issue' label for both the client and server repos. Any plans to add that label soon?

Reply:
changed to `unconfirmed`